CMakeLists.txt

```diff
@@ -194,10 +194,6 @@ endif ()
 set(ENTERPRISE_INCLUDE_DIR "enterprise")
 option(USE_ENTERPRISE "enable enterprise build" OFF)
 
-# we want the following definitions to be in effect for both rocksdb and arangodb
-add_definitions("-DNROCKSDB_THREAD_STATUS")
```

Comment:
note: the new version of RocksDB still checks for a define `NROCKSDB_THREAD_STATUS`, but if it is set, then it does not link anymore. So it seems that the option is now unsupported and we shouldn't set it anymore. Thus removed it here.